docs/web/docs/3-reference/3-schemas/awsaccount.json

```diff
@@ -1,194 +0,0 @@
-{
```

Comment:
is there some intentional removal. I didn't quite understand why all these are gone now.